tests/bench/locomo_baseline.py

```
@@ -0,0 +1,534 @@
+                        "passed": passed,
+                    }
+
+                    result["question_results"].append(question_result)
+
+                    status = "PASS" if passed else "FAIL"
+                    print(f"    [{status}]")
+                    if not passed:
+                        print(f"      Expected: {expected_answer}")
+                        print(f"      Got: {actual_response[:200]}...")
+
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)